posts/src/components/Modal.jsx

```diff
@@ -0,0 +1,23 @@
+import React, {useState, useEffect} from 'react';
+import Modal from "react-bootstrap/Modal";
+import Button from "react-bootstrap/Button";
+
+function AuthorModal({showModal, user, closeModal}) {
+    return (
+        <Modal show={showModal}>
```

Comment:
to prevent console errors, when clicking outside of an open modal, add:
```suggestion
        <Modal show={showModal} onHide={closeModal}>
```


https://user-images.githubusercontent.com/31325372/115244673-d474f600-a12c-11eb-91bc-7e3ca3f18cae.mov